assignment3/magic_dict.py

```diff
@@ -0,0 +1,7 @@
+
+class Magic_dictionary:
```

Comment:
Suggestion #5 is interesting. I suppose the dictionary could be very long. The grid might also be really big though. Which one should we be more afraid of?